README.md

```diff
@@ -777,6 +777,43 @@ try{
 ```
 
 
+#### Update contract transaction
+The following example demonstrates how to construct a "updateContract" transaction, which is used to update a smart contract module.
+
+ ```java
+try{
+    byte[] paramBytes = new byte[0];
+    TransactionSigner signer = TransactionSigner.from(
+        SignerEntry.from(Index.from(0), Index.from(0),
+            firstSecretKey),
+        SignerEntry.from(Index.from(0), Index.from(1),
+            secondSecretKey));
+
+    val transaction = TransactionFactory.newUpdateContract()
```

Comment:
I don't think the README/docs should be using `Lombok` features (like `val`).

Reply:
done